client/src/main/java/io/pravega/client/admin/impl/ReaderGroupManagerImpl.java

```diff
@@ -47,7 +48,8 @@
 
     public ReaderGroupManagerImpl(String scope, URI controllerUri, ConnectionFactory connectionFactory) {
         this.scope = scope;
-        this.controller = new ControllerImpl(controllerUri);
+        this.controller = new ControllerImpl(controllerUri, ControllerImplConfig.builder().build(),
```

Comment:
The Controller client will not shutdown externally supplied executors. I guess that's the convention anyway - the class which creates it should be responsible to shut it down. I will add appropriate comments in the code